controller/eks-cluster-config-handler.go

```diff
@@ -444,7 +449,17 @@ func (h *Handler) create(config *eksv1.EKSClusterConfig, sess *session.Session,
 
 	displayName := config.Spec.DisplayName
 
-	var err error
+	listOutput, err := eksService.ListClusters(&eks.ListClustersInput{})
+	if err != nil {
+		return config, fmt.Errorf("error listing clusters: %v", err)
+	} else {
+		for _, cluster := range listOutput.Clusters {
+			if cluster != nil && *cluster == displayName {
+				return config, fmt.Errorf("cannot create cluster with same name as an existing cluster")
```

Comment:
you should mention which cluster this is referencing
```suggestion
				return config, fmt.Errorf("cannot create cluster [%s] because a cluster exists in EKS with that name")
```